src/Accounts/Accounts/Environment/AddAzureRMEnvironment.cs

```diff
@@ -163,6 +163,10 @@ public string DataLakeAudience
            HelpMessage = "The endpoint to use when communicating with the Azure Log Analytics API.")]
```

Comment:
Need to add a line in the changelog.md for Az.Accounts to describe this change - thsi is the trigger for changing the version and releasing a new version fo the module

Reply:
Updated ChangeLogs